financial-services-accelerator/accelerators/fs-is/repository/resources/financial-services.yaml

```diff
@@ -24,6 +24,32 @@ components:
         applicable_params:
           - "Data.ExpirationDateTime"
 
+    consentDataRetrievalPolicy: &consentDataRetrievalPolicy
+      name: "Consent JSON Build Policy"
```

Comment:
This logic anyway need for all the specs/use-cases,can we remove this as enabling a policy and keep as a mandatory step b4 executing rest of the policies. Else someone can remove this and destruct the flow.

Reply:
Added a git issue to track this.
https://github.com/wso2/financial-services-accelerator/issues/358